src/style.css

```diff
@@ -0,0 +1,107 @@
+@font-face {
+  font-family: “Avenir”, sans-serif;
+  font-weight: 300;
+  font-style: normal;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: “Avenir”, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.searchBar1 {
+  box-sizing: border-box;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  height: 70px;
+  width: 100%;
+  margin-top: 20px;
+  display: flex;
+}
+
+.firstSearchBar {
+  box-sizing: border-box;
+  width: 100%;
+  border: none;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+  border-width: none;
+  outline: none;
+  /* padding-right: 709px; */
+  padding-left: 62px;
+}
+
+.image1 {
+  width: 19px;
+  height: 19px;
+  background: none;
+  padding: 25px 17px 25px 26px;
+  position: absolute;
+}
+
+.searchBar1:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.firstSearchBar:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
+
+.searchBar2 {
+  box-sizing: border-box;
+  width: 100%;
+  height: 42px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  margin-top: 20px;
+  display: flex;
+}
+
+.secondSearchBar {
+  box-sizing: border-box;
+  width: 100%;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 19px;
+  color: #3d4e61;
+  border-width: none;
+  border: none;
+  outline: none;
+  /* padding-right: 226.71px; */
+  padding-left: 33px;
+}
+
+.image2 {
+  width: 11px;
+  color: #687c94;
+  padding: 15px 16px 13px;
+  display: flex;
+  position: absolute;
+}
+
+.searchBar2:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.secondSearchBar:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
```

Comment:
[STYLES] - if you have two or more similar elements with portions of similar styles with different values - use one of the elements as the basic case, and override necessary styles for other cases. Explanation: The point is not in the names of the classes, the point is: when there are several similar elements, ex., 2 inputs, for one we can give a class input, for example, and for the second - input input--small. We write all the styles for .input, but for .input-small we write only those styles that differ in design, and we need this second input to look a little different. Element with class .input without extra classes should also look like a full-fledged styled element.